portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -32,24 +32,27 @@
 @WebServlet("/login")
 public class LoginServlet extends HttpServlet {
 
+  private String urlToRedirectToAfterUserLogsIn = "/login";
```

Comment:
Since it's the same, you can probably use the same variable for both. If it ever becomes different, then you can create separate ones. 

Also a static variable might be better here, wdyt Peter? 

`private static final String REDIRECT_URL = "/login";`

Reply:
Fixed